cdn-uploader.js

```diff
@@ -0,0 +1,45 @@
+const ftp = require("basic-ftp")
+
+const config = {
+host: process.env.AKAMAI_CDN_HOSTNAME,
+user: process.env.AKAMAI_CDN_USERNAME,
+password: process.env.AKAMAI_CDN_PASSWORD,
+cpCode: process.env.AKAMAI_CDN_CPCODE,
+port: 21,
+basePath: "./builds"
+}
```

Comment:
It seems that in general, you are using semicolon (`;`), could you use them in all required places?

Reply:
To keep a consistent style.

Reply:
Yes I will do it